Apps/APN-UIKit/APN UIKit/View/LoginViewController.swift

```diff
@@ -92,9 +92,13 @@ class LoginViewController: BaseViewController {
         if let name = firstNameTextField.text, !name.isEmpty {
             body = ["first_name": name]
         }
-        CustomerIO.shared.identify(identifier: emailId, body: body)
         storage.userEmailId = emailId
         loginRouter?.routeToDashboard()
+        guard let data = body else {
+            CustomerIO.shared.identify(identifier: emailId)
+            return
+        }
+        CustomerIO.shared.identify(identifier: emailId, body: data)
```

Comment:
This fixes a crash that happens when this method sends a `nil` data to `identify` that CDP doesn't expect.

Reply:
This can be done in a separate PR as this is unrelated to this ticket.